hack/Dockerfile

```
@@ -283,7 +296,75 @@ ENTRYPOINT ["pyenv", "exec", "python3", "-m", "gunicorn", "--config=/src/src/das
 
 # -----------------------------------------------------------------------------
 
-FROM base AS archivematica-tests
+FROM base-builder as archivematica-tests
+
+ARG USER_ID=1000
+ARG GROUP_ID=1000
+ARG PYENV_DIR=/pyenv
```

According to the docs, we don't need to redefine the default again when pulling the argument, e.g.:

```suggestion
ARG USER_ID
ARG GROUP_ID
ARG PYENV_DIR
```

---

Thanks!